code/modules/hydroponics/hydroponics.dm

```diff
@@ -233,7 +233,7 @@
 					harvest = 1
 				else
 					lastproduce = age
-			if(prob(5))  // On each tick, there's a 5 percent chance the pest population will increase
+			if(prob(1) || (myseed.get_gene(/datum/plant_gene/trait/plant_type/carnivory) && prob(5)))  // 1% chance for pests on most plants, 5% on carnivorous plants since they attract bugs.
```

Comment:
this might not be huge but carnivorous plants roll 2 probs which the comment doesn't suggest